src/DDTrace/Integrations/SymfonyMessenger/DDTraceStamp.php

```diff
@@ -1,20 +0,0 @@
-<?php
-
-namespace DDTrace\Integrations\SymfonyMessenger;
-
-use Symfony\Component\Messenger\Stamp\StampInterface;
-
-final class DDTraceStamp implements StampInterface
-{
-    private $headers;
-
-    public function __construct(array $headers)
-    {
-        $this->headers = $headers;
-    }
-
-    public function getHeaders(): array
-    {
-        return $this->headers;
-    }
-}
```

Comment:
Please do not delete this, or a rolling update of the extension will break if the receiver is updated first.

Mark it simply as deprecated.